auxx/src/Plugin.hs

```diff
@@ -24,8 +24,8 @@ import           System.IO (hFlush, stdout)
 import           System.Wlog (CanLog, HasLoggerName, WithLogger, logDebug, logInfo)
 
 import           Pos.Communication (Conversation (..), OutSpecs (..), SendActions (..),
-                                    WorkerSpec, delegationRelays, relayPropagateOut, txRelays,
-                                    usRelays, worker)
+                                   WorkerSpec, delegationRelays, relayPropagateOut,
+                                   txRelays, usRelays, worker)
```

Comment:
It's a strange change, maybe you misconfigured `stylish-haskell` or something 🤷‍♂️

Reply:
I didn't load the stylish-haskell at all. Now I have set it to re-format on save.

Reply:
And now something is really wrong. There should be no changes in this file at all, but you changed formatting of all importsn
Apparently your `stylish-haskell` is misconfigured. We have proper stylish-haskell config in the root of the repository. However, it requires version 0.8.1.0. Probably your stylish-haskell is older. In this case please update. Otherwise we need to figure out what's wrong with your setup.